### PR TITLE
add developer survey notification

### PR DESCRIPTION
**Pull request by StephenFluin**

### CODE REVIEW

**Comment by StephenFluin** [comment]
@dmitrydao What do you think?

File: _includes/main.liquid [hunk 1]
```diff
@@ -60,6 +61,14 @@
     </head>
 
     <body class="body">
+        <div id="notification" key="dev-survey-2021">
+            <div class="notification-info">
+                <span id="notification-close">X</span>
+                <span class="notification-text">
+                    Help Chainlink by taking a <b><a href="https://chainlinkcommunity.typeform.com/to/V2z25k8N">1 minute survey</b></a>!
```
Comment (dmitrydao) [inline _includes/main.liquid:68]: I think the link should be this one: https://chn.lk/dev-survey-2021

Comment (dmitrydao) [reply _includes/main.liquid:68]: > @dmitrydao What do you think?

Love it overall. It doesn't appear on the page reload though. I assume this is the expected behavior, right?

Comment (StephenFluin) [reply _includes/main.liquid:68]: Sounds good. I'm get that fixed up. How's the functionality?

Comment (dmitrydao) [reply _includes/main.liquid:68]: I think it's great, especially the bouching effect.

Comment (StephenFluin) [reply _includes/main.liquid:68]: It doesn't appear or doesn't bounce? Expected behavior is

1. First load, it slides in to draw attention
2. Future loads, simply present
3. X it out, never see it again

Comment (dmitrydao) [reply _includes/main.liquid]: It doesn't appear when you close it but that is the expected behavior so I think we're good.

**Review by alexroan — APPROVED** [review]

**Pull request merged**